understanding/character-key-shortcuts.html

```diff
@@ -55,7 +55,7 @@ <h1>達成基準 2.1.4: 文字キーのショートカットを理解する</h1>
          <section id="intent">
             <h2>意図</h2>
             <p>この達成基準の意図は、キーボードショートカットの誤った動作を減らすことである。文字キーのショートカットは、多くのキーボード利用者にはうまく機能するが、(入力手段がひとつずつの文字列である) 音声入力の利用者や、誤ってキーを押す傾向にあるキーボード利用者にとっては不適切かつ、いら立たしい。この問題を是正するためには、コンテンツ制作者は、利用者が文字キーのみで構成されるショートカットを解除する、又は再構成できるようにする必要がある。</p>
-            <p>この達成基準は、リストボックスやドロップダウンメニューなどのコンポーネントには影響しないことに注意すること。このコンポーネントは一つ以上の文字キーで選択されるかもしれない値 (単語) を含むが、ショートカットがコンポーネントにフォーカスされている時のみ有効となる。メニューなどの他のコンポーネントは、単一の文字キーを押して項目を選択する前に、単一の非文字のショートカット (例:ALT や ALT+F) でアクセス又は開くことができる。これにより、メニューを起動するための完全なパスが非印字文字を含む 2 段階ショートカットになる。 <a href="https://www.w3.org/TR/html5/single-page.html#the-accesskey-attribute">Accesskey</a> は修飾キーを含んでいるため、影響を受けない。</p>
+            <p>この達成基準は、リストボックスやドロップダウンメニューなどのコンポーネントには影響しないことに注意すること。このコンポーネントは一つ以上の文字キーで選択されるかもしれない値 (単語) を含むが、ショートカットがコンポーネントにフォーカスされている時のみ有効となる。メニューなどの他のコンポーネントは、単一の文字キーを押して項目を選択する前に、単一の非文字のショートカット (例 :ALT や ALT+F) でアクセス又は開くことができる。これにより、メニューを起動するための完全なパスが非印字文字を含む 2 段階ショートカットになる。 <a href="https://www.w3.org/TR/html5/single-page.html#the-accesskey-attribute">Accesskey</a> は修飾キーを含んでいるため、影響を受けない。</p>
```

Comment:
例 :ALT → 例: ALT でしょうか。

```suggestion
            <p>この達成基準は、リストボックスやドロップダウンメニューなどのコンポーネントには影響しないことに注意すること。このコンポーネントは一つ以上の文字キーで選択されるかもしれない値 (単語) を含むが、ショートカットがコンポーネントにフォーカスされている時のみ有効となる。メニューなどの他のコンポーネントは、単一の文字キーを押して項目を選択する前に、単一の非文字のショートカット (例: ALT や ALT+F) でアクセス又は開くことができる。これにより、メニューを起動するための完全なパスが非印字文字を含む 2 段階ショートカットになる。 <a href="https://www.w3.org/TR/html5/single-page.html#the-accesskey-attribute">Accesskey</a> は修飾キーを含んでいるため、影響を受けない。</p>
```